docs/Targets.md

```diff
@@ -409,6 +410,82 @@ add-symbol-file test-app/image.elf 0x08008100
 mon reset init
 ```
 
+## STM32C0
+
+Supports STM32C0x0/STM32C0x1.
+
+Example and instructions are for the STM Nucleo-C031C6 dev board using the 
+STM32Cube 
+
+Example 32KB partitioning on STM32-G070:
+
+- Sector size: 2KB
+- Wolfboot partition size: 14KB
+- Application partition size: 8 KB
+- Swap size 2KB
+
+```C
+#define WOLFBOOT_SECTOR_SIZE                 0x800   /* 2 KB */
+#define WOLFBOOT_PARTITION_BOOT_ADDRESS      0x08004000 /* offset 16kB to 24kB */
+#define WOLFBOOT_PARTITION_SIZE              0x2000  /* 8 KB */
+#define WOLFBOOT_PARTITION_UPDATE_ADDRESS    0x08006000 /* offset 24kB to 32kB */
+#define WOLFBOOT_PARTITION_SWAP_ADDRESS      0x08003800 /* offset 14kB to 16kB */
```

Comment:
I assume you resolved the issue with using the end of flash and it was related to `NVM_FLASH_WRITEONCE`?

Reply:
Can't prove that yet.  One of the things I need to test.  I'll make a maximum-sized test app to prove it out.